packages/components/input-phone/index.js

```diff
@@ -104,6 +108,13 @@ const InputPhone = ({
   const Row = ({ data, index, style }) => {
     const country = data[index];
     const prefix = "+";
+    const RtlRowComponent = () => (
```

Comment:
Why is this block extracted to a constant and the other is not?

Reply:
created a component for the second condition